### PR TITLE
RDM-9024 CICD Demo

### PR DESCRIPTION
**Pull request by hemantt**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by mario-paniccia — APPROVED** [review]
👍

File: excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ComplexFieldTypeParser.java [hunk 1]
```diff
@@ -118,6 +127,8 @@ private ComplexFieldEntity parseComplexField(DefinitionDataItem definitionDataIt
         complexField.setOrder(order);
         complexField.setShowCondition(parseShowCondition(definitionDataItem.getString(ColumnName.FIELD_SHOW_CONDITION)));
         complexField.setDisplayContextParameter(definitionDataItem.getString(ColumnName.DISPLAY_CONTEXT_PARAMETER));
+        complexField.setSearchable(definitionDataItem.getBooleanOrDefault(ColumnName.SEARCHABLE, true));
```
Comment (RebeccaBaker) [inline excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ComplexFieldTypeParser.java:130]: this looks like its was added by a merge from develop - not part of RDM-9024

File: repository/src/main/resources/db/changelog/db.changelog-master.xml [hunk 2]
```diff
@@ -88,4 +88,7 @@
     <include file="db/changelog/db.changelog-rdm-8382.xml"/>
     <include file="db/changelog/db.changelog-rdm-8509.xml"/>
     <include file="db/changelog/db.changelog-rdm-8692.xml"/>
+    <include file="db/changelog/db.changelog-rdm-2676.xml"/>
+    <include file="db/changelog/db.changelog-rdm-8869.xml"/>
```
Comment (RebeccaBaker) [inline repository/src/main/resources/db/changelog/db.changelog-master.xml:92]: this looks like its was added by a merge from develop - not part of RDM-9024

File: excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ComplexFieldTypeParser.java (hunk 1 above)
Comment (hemantt) [reply excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ComplexFieldTypeParser.java]: Thanks @RebeccaBaker, that line is commented out now

File: repository/src/main/resources/db/changelog/db.changelog-master.xml (hunk 2 above)
Comment (hemantt) [reply repository/src/main/resources/db/changelog/db.changelog-master.xml]: Thanks @RebeccaBaker, log updated

**Review by RebeccaBaker — APPROVED** [review]

**Review by RebeccaBaker — APPROVED** [review]

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
wiring seems to be not aligned for consuming everything from aat.

**Pull request closed without merging**